models/netsuite2/intermediate/base/int_netsuite2__consolidated_exchange_rates.sql

```diff
@@ -1,4 +1,4 @@
-{{ config(enabled=var('netsuite_data_model', 'netsuite') == var('netsuite_data_model_override','netsuite2')) }}
+{{ config(enabled=(var('netsuite_data_model', 'netsuite') == var('netsuite_data_model_override','netsuite2') and var('netsuite2__using_accounting_book_subsidiaries', true))) }}
```

Comment:
all this models does is join in a field from `accounting_book_subsidiaries`, so we can dynamically reference the base staging table instead of this model

Reply:
an alternative is to just add a `null` accounting_book_id field if the `accounting_book_subsidiaries` is disabled, but then this table would kinda be made for no reason